pkgs/top-level/all-packages.nix

```diff
@@ -8204,7 +8315,7 @@ in
     stdenv = if stdenv.cc.isClang then llvmPackages_5.stdenv else stdenv;
   };
 
-  me_cleaner = pythonPackages.callPackage ../tools/misc/me_cleaner { };
+  me_cleaner = python2Packages.callPackage ../tools/misc/me_cleaner { };
```

Comment:
Some of these packages probably already works in python3. For example me_cleaner does.

Reply:
You are right, a lot of the packages probably work with python3 though I consider that out-of-scope for this PR.